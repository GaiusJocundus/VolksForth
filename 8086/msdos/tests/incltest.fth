
include log2file.fb
logopen incltest.log

.( hello, world) cr
: test-hello ." hello, world, from test-hello" cr ;
test-hello

logclose
