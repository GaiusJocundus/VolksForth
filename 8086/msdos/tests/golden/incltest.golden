hello, world
hello, world, from test-hello
